app/build.gradle

```diff
@@ -422,6 +426,10 @@ android {
         // MetaStore only apply to oculusvr builds.
         if (store == 'metaStore' && !platform.startsWith('oculusvr'))
             variant.setIgnore(true);
+
+        // MagicLeap only for AOSP builds.
+        if (store == 'magicleap' && !platform.startsWith('aosp'))
```

Comment:
You can even restrict it more by specifying x64 abi

Reply:
Replaced with:

        // MagicLeap only for AOSP x64 builds.
        if (store == 'magicleap' && (platform != 'aosp' || abi != 'x64'))
            variant.setIgnore(true);